integ-test/src/test/java/org/opensearch/sql/correctness/runner/resultset/DBResult.java

```diff
@@ -191,4 +191,22 @@ private static <T extends Comparable<T>> List<T> sort(Collection<T> collection)
     return list;
   }
 
+  public boolean equals(final Object o) {
+    if (o == this) {
+      return true;
+    }
+    if (!(o instanceof DBResult)) {
+      return false;
+    }
+    final DBResult other = (DBResult) o;
+    // H2 calculates the value before setting column name
+    // for example, for query "select 1 + 1" it returns a column named "2" instead of "1 + 1"
+    boolean skipColumnNameCheck = databaseName.equalsIgnoreCase("h2") || other.databaseName.equalsIgnoreCase("h2");
+    if ((!skipColumnNameCheck && !schema.equals(other.schema) || (skipColumnNameCheck &&
```

Comment:
a little complex to understand.  It would be best to break this into parts.  Something like: 
```
if (!skipColumnNameCheck && !schema.equals(other.schema)) {
  return false;
}
if (skipColumnNameCheck && !schema.stream().map(Type::getType).collect(Collectors.toList())
                .equals(other.schema.stream().map(Type::getType).collect(Collectors.toList()))) {
  return false;
}
```

Reply:
Thanks, fixed in 4e51282f7.